src/metabase/driver/vertica.clj

```diff
@@ -0,0 +1,218 @@
+
+(ns metabase.driver.vertica
+  ;; TODO - rework this to be like newer-style namespaces that use `u/drop-first-arg`
+  (:require [clojure.java.jdbc :as jdbc]
+            (clojure [set :refer [rename-keys], :as set]
+                     [string :as s])
+            [clojure.tools.logging :as log]
+            [honeysql.core :as hsql]
+            [metabase.db.spec :as dbspec]
+            [metabase.driver :as driver]
+            [metabase.driver.generic-sql :as sql]
+            [metabase.util :as u]
+            [metabase.util.honeysql-extensions :as hx]))
+
+(defn- column->base-type
+  "Map of Vertica column types -> Field base types.
+   Add more mappings here as you come across them."
+  [column-type]
+  ({:Boolean        :type/Boolean
+    :Integer        :type/Integer
+    :Bigint         :type/BigInteger
+    :Varbinary      :type/*
+    :Binary         :type/*
+    :Char           :type/Text
+    :Varchar        :type/Text
+    :Money          :type/Decimal
+    :Numeric        :type/Decimal
+    :Double         :type/Decimal
+    :Float          :type/Float
+    :Date           :type/Date
+    :Time           :type/Time
+    :TimeTz         :type/Time
+    :Timestamp      :type/DateTime
+    :TimestampTz    :type/DateTime
+    :AUTO_INCREMENT :type/Integer
+    (keyword "Long Varchar")    :type/Text
+    (keyword "Long Varbinary")  :type/*}
+   column-type))
+
+(defn- column->special-type
+  "Attempt to determine the special-type of a Field given its name and Postgres column type."
+  [column-name column-type]
+  ;; this is really, really simple right now.  if its postgres :json type then it's :json special-type
+  (when (= column-type :json)
+    :json))
+
+(def ^:const ssl-params
+  "Params to include in the JDBC connection spec for an SSL connection."
+  {:ssl        true
+   :sslmode    "require"
+   :sslfactory "org.postgresql.ssl.NonValidatingFactory"})  ; HACK Why enable SSL if we disable certificate validation?
```

Comment:
Good catch! The way I got the Vertica driver is by making a duplicate of the Postgres driver and debugging until it started working. I would guess there are a few things in the Vertica driver that don't need to be there, like this.

I removed this and a few other references to postgres stuff.